int32 mode
int32 DISABLED = 0
int32 POSITION = 1
int32 EFFORT = 2
int32 HOLD_POSITION = 3

float64 set_point
